pages/common/musescore.md

```diff
@@ -0,0 +1,32 @@
+# musescore
+
+> MuseScore 3 sheet music editor.
+> More information: <https://musescore.org/en/handbook/3/command-line-options>.
+
+- Use a specific audio driver:
+
+`musescore --audio-driver {{jack|alsa|portaudio|pulse}}`
+
+- Set the MP3 output bitrate in kbit/s:
+
+`musescore --bitrate {{bitrate}}`
+
+- Start MuseScore in debug mode:
+
+`musescore --debug`
+
+- Enable experimental features, such as layers:
+
+`musescore --experimental`
+
+- Export the given file to the specified output file. The file type depends on the extension given:
```

Comment:
Is this sentence wrong? @bl-ue 
```suggestion
- Export the given file to the specified output file. The file type depends on the given extension:
```

Reply:
@navarroaxel no, both versions are correct and convey the exact same thing.  It doesn't matter which one we use.  I suppose we should pick the one that's easier for you to read because that likely means easier for many others.